src/neo-vm/ExecutionEngine.cs

```diff
@@ -289,7 +305,32 @@ private bool ExecuteInstruction()
                     }
                 case OpCode.THROW:
                     {
-                        return false;
+                        if (!TryPop(out StackItem error)) return false;
+                        return ExecuteThrow(error);
+                    }
+                case OpCode.TRY:
+                    {
+                        int catchOffset = instruction.TokenI8;
+                        int finallyOffset = instruction.TokenI8_1;
+                        return ExecuteTry(catchOffset, finallyOffset);
+                    }
+                case OpCode.TRY_L:
```

Comment:
Shall we use the neo format for avoid all the `_L` ? or it's better to have fixed sizes per opcode? 🤔 

https://github.com/neo-project/neo/blob/a056f256fe0b5e4bdfc697d50ecf1e23324c0b2d/src/neo/IO/Helper.cs#L223-L237

Reply:
I prefer fixed size.